packages/manager-components/src/components/content/headers/headers.component.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+import { Description, Title, Subtitle } from '../../typography';
+
+export interface HeadersProps {
+  title?: string;
+  subtitle?: string;
+  description?: string;
+  hasHeaderButton?: React.ReactElement;
```

Comment:
The prefix 'has' indicates a boolean
Just `HeaderButton` or use `children` to be less specific